README.md

```
@@ -0,0 +1,1136 @@
+      <td>기타사항</td>
+      <td>Notion, Google Drive, Google Sheets, Figma (UX 시안)</td>
+    </tr>
+  </tbody>
+</table>
+
+
+---
+
+## **💡2. 팀원 소개**
+
+| [<img src="https://github.com/Dubabbi.png" width="150px">](https://github.com/Dubabbi) | [<img src="https://github.com/kcw9609.png" width="150px">](https://github.com/kcw9609) | [<img src="https://github.com/20210699.png" width="150px">](https://github.com/20210699) | [<img src="https://github.com/yuchaemin2.png" width="150px">](https://github.com/yuchaemin2) | [<img src="https://github.com/yooni1231.png" width="150px">](https://github.com/yooni1231) |
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)